README.md

```diff
@@ -184,37 +184,41 @@ Further details can be found in the ITM Server FAQ below.
     Events communicate information from the scenario to ADM.  Each event has unstructured text and one of the following event types:
-    * change: signifies a change in state from one known value to another.
-      * source is the entity telling of the state change
-      * relevant_state is/are the state that changed
-    * emphasize: point out a previously known value from the state that has not changed.
-      * source is the entity emphasizing the current state
-      * relevant_state is/are the current state that is being emphasized
-    * inform: notify of new state or state that has gone from an unknown value to a known value.
-      * source is the entity informing of the state
-      * relevant_state is/are the state that is new or newly known
-    * order: authoritarian directive, usually to perform an action.
-      * source is the entity ordering the action
-      * action_id is the ID of the ordered action; the action_id matches an action from `get_available_state`
-    * recommend: suggestion or opinion, usually to perform an action.
-      * source is the entity recommending the action
-      * action_id is the ID of the ordered action; the action_id matches an action from `get_available_state`
+    * `change`: signifies a change in state from one known value to another.
+      * `source` is the entity telling of the state change
+      * `when` indicates when (in minutes) the state changed (negative value) or is expected to change (positive value)
+      * `relevant_state` is/are the state that changed
+    * `emphasize`: point out a previously known value from the state that has not changed.
+      * `source` is the entity emphasizing the current state
+      * `relevant_state` is/are the current state that is being emphasized
+    * `inform`: notify of new state or state that has gone from an unknown value to a known value.
+      * `source` is the entity informing of the state
+      * `when` indicates when (in minutes) the state happened (negative value) or is expected to happen (positive value)
+      * `relevant_state` is/are the state that is new or newly known
+    * `order`: authoritarian directive, usually to perform an action.
+      * `source` is the entity ordering the action
+      * `action_id` is the ID of the ordered action; the action_id matches an action from `get_available_state`
+    * `recommend`: suggestion or opinion, usually to perform an action.
+      * `source` is the entity recommending the action
+      * `action_id` is the ID of the ordered action; the action_id matches an action from `get_available_state`
+
     The `relevant_state` property is a list of string paths within the `State` object, in which indexed lists are context-sensitive:
-      * for an aid or a character, it's the id
-      * for a supply, it's the type
-      * for a threat, it's the threat_type
-      * for an injury, it's the location
+    * for an `Aid` or a `Character`, it's the `id`
+    * for a `Supplies`, it's the `type`
```

Comment:
I just want to make sure you intended "a `Supplies`"

Reply:
I did it consciously, but I think it's more readable / better English just to remove the a/an in all cases.  Thanks.